addon/ng2/blueprints/mobile/files/__path__/main-app-shell.ts

```diff
@@ -0,0 +1,35 @@
+import {provide} from '@angular/core';
+// import {ROUTER_PROVIDERS} from '@angular/router-deprecated';
```

Comment:
I'll actually remove them for now since I still have to sort out the router upgrade.